docs/components/card.md

```diff
@@ -1,275 +1,179 @@
-
-<KCard body="I am the body."/>
-
-```html
-<KCard body="I am the body."/>
-```
-
-### borderVariant
-
-Sets top border or no border. If neither set default will have border
-
-- `borderTop`
-- `noBorder`
-
-<div class="borderless-cards">
-  <KCard title="Card without border" body="Body Content" border-variant="noBorder"/>
-
-  <KCard title="Card with top border" body="Body Content" border-variant="borderTop"/>
-</div>
-
-```html
-<KCard title="Card without border" body="Body Content" border-variant="noBorder"/>
-
-<KCard title="Card with top border" body="Body Content" border-variant="borderTop"/>
-```
+## Props
 
-### hasHover
+### title
 
-Set if card should have shadow state (shadow) on hover
+String to be used as a title. Can also be [slotted](#slots).
 
-<KCard title="hasHover" body="This card only has a shadow on hover" has-hover />
+### content
 
-```html
-<KCard title="hasHover" body="This card only has a shadow on hover" has-hover />
-```
+String to be used as content. Can also be [slotted](#default).
 
-### hasShadow
+<KCard
+  content="Card content"
+  title="Card Title"
+/>
 
-Set if the card should always have shadow state (shadow)
+## Slots
 
-<KCard title="hasShadow" body="This card always has a shadow" has-shadow />
+### title
 
-```html
-<KCard title="hasShadow" body="This card always has a shadow" has-shadow />
-```
+Content to be rendered in the title of the card. For example refer to [`footer` slot](#footer).
 
-## Using flex box
+:::warning NOTE
+To preserve a valid HTML structure, avoid slotting in block-level elements such as a `div` into the `title` slot as it will be rendered inside a `h4` element.
+:::
 
-Cards can be arranged with flex box.
+### actions
 
-<div class="card-flex-container">
-  <KCard
-    title="Left"
-    body="This card only has a title"
-  />
-  <KCard
-    title="Center"
-    body="This card always has a icon button"
-  >
-    <template v-slot:actions>
-      <KButton size="small" appearance="secondary">
-        <KIcon
-          icon="gearFilled"
-          size="16"
-          view-box="0 0 16 16"
-        />
-      </KButton>
-    </template>
-  </KCard>
-  <KCard
-    title="Right"
-    body="This card always has a button"
-  >
-    <template v-slot:actions>
-      <KButton size="small" appearance="secondary">View All</KButton>
-    </template>
-  </KCard>
-</div>
+Located next to card title, this slot serves for slotting in any action elements. For example refer to [`footer` slot](#footer).
 
-```html
-<KCard title="Left" body="This card only has a title" />
-<KCard title="Center" body="This card always has a icon button">
-  <template v-slot:actions>
-    <KButton>
-      <KIcon
-        icon="gearFilled"
-        size="16"
-        view-box="0 0 16 16"
-      />
-    </KButton>
-  </template>
-</KCard>
-<KCard title="Right" body="This card always has a button">
-  <template v-slot:actions>
-    <KButton>View All</KButton>
-  </template>
-</KCard>
+### default
```

Comment:
suggestion: can we make it a standard to document the default slot first?

Reply:
Fixed.